pyproject.toml

```diff
@@ -88,7 +88,7 @@ update_cli_docs = "python ./scripts/generate_reference_docs.py"
 shell = "pytest -n auto tests/e2e"
 
 [tool.poe.tasks.test_integration]
-shell = "pytest -n auto scripts tests/integration"
```

Comment:
Not in scope of the PR

Reply:
I was debugging failing tests on CI only...